net/pfSense-pkg-frr/files/usr/local/pkg/frr/frr_bgp_neighbors.xml

```diff
@@ -375,6 +375,48 @@
 			<source_value>value</source_value>
 		</field>
 
+		<field>
+			<name>BFD</name>
+			<type>listtopic</type>
+		</field>
+		<field>
+			<fielddescr></fielddescr>
+			<description>
+				<![CDATA[
+				<b>BFD Enable</b> : Listen for BFD events registered on the same target as this BGP neighbor.
+				When BFD peer goes down it immediately asks BGP to shutdown the connection with its neighbor and, when it goes back up, notify BGP to try to connect to it.
```

Comment:
This sentence needs reworded. It's missing words/has, awkward syntax, and seems like it could be shortened significantly.

Reply:
This is straight from the FRR docs, I've reworded as best I can.